src/emulator.js

```diff
@@ -1,11 +1,234 @@
+import 'babel-polyfill';
+import 'source-map-support/register';
```

Comment:
No need for `babel-polyfill` and `source-map-support` when it's in the `index.js`.